actors/migration/nv15/miner.go

```diff
@@ -216,26 +216,21 @@ func (m minerMigrator) migrateDeadlines(ctx context.Context, store adt.Store, de
 
 	var outDeadlines miner7.Deadlines
```

Comment:
Several limes up, we can remove the `&& sectors == m.emptySectorsV7`. If the deadlines are empty, we don't need to do anything.